assets/styles/main.css

```diff
@@ -0,0 +1,3 @@
+html {
+    font-family: 'Calibri';
```

Comment:
you should always enter multiple fonts as I doubt that any computer contains the font ``Calibri`` and if it doesn't exist, the default font will be used which causes an odd markup.
I'd recommend sth. like this: ``Calibri, Verdana, sans-serif``